test/lib/usd/translators/UsdExportGeomSubsetTest/UsdExportGeomSubsetTest.ma

```diff
@@ -0,0 +1,348 @@
+//Maya ASCII 2021ff01 scene
+//Name: ExportGeomSubsetTest.ma
+//Last modified: Wed, Oct 21, 2020 10:54:20 AM
+//Codeset: 1252
+requires maya "2021ff01";
```

Comment:
Similar comment as in another PR, is this ok when running the test against Maya 2019 or Maya 2020?